public/app/features/alerting/unified/components/contact-points/ContactPoints.test.tsx

```diff
@@ -22,6 +25,16 @@ import setupVanillaAlertmanagerFlavoredServer, {
 import { RECEIVER_META_KEY } from './constants';
 import { ContactPointWithMetadata, ReceiverConfigWithMetadata, RouteReference } from './utils';
 
+/**
+ * Flushes out microtasks so we don't get warnings from @floating-ui/react
+ * as per https://floating-ui.com/docs/react#testing
+ */
+const flushMicrotasks = async () => {
```

Comment:
maybe we could move this function to an utility folder? I guess we would need it in several places?

Reply:
I've moved this to a utils file and combined it with the other `testWithFeatureToggles` method etc that I added